app/src/services/object.js

```diff
@@ -230,7 +231,8 @@ const service = {
         path: data.path,
         public: data.public,
         active: data.active,
-        updatedBy: data.userId ?? SYSTEM_USER
+        updatedBy: data.userId ?? SYSTEM_USER,
+        lastSyncedDate: new Date()
```

Comment:
i think this needs to be ` lastSyncedDate: data.lastSyncedDate`
and the value needs to come in as a parameter. if not provided it will be undefined and show as null in the db.
because there are places in the app where we call this update method but we're not actually syncing with all the data in S3.

Reply:
This line is disingenuous and should not be included in the patch operation. The intent of the lastSyncedDate is to record the last time an _explicit_ synchronization operation was applied to the object. When an object is updated via COMS, you have no guarantee that COMS state is completely synchronized with S3. The most you can guarantee is that the latest new version uploaded exists in both COMS and S3. In the event S3 and COMS has already diverged prior to the update, (let's say 3 versions known in COMS, but S3 has 5 versions), after this operation, COMS still won't know about the other two versions without a formal sync. We should not be populating it here.

Reply:
after further discussion, i agree with Jeremy, the lastSyncDate should be null when an object is created or updated.
We only want to populate this field with the current timestamp when the syncObject() function is called (services/sync.js)